relay-server/src/actors/healthcheck.rs

```diff
@@ -36,7 +33,7 @@ impl Healthcheck {
     ///
     /// Panics if the service was not started using [`Healthcheck::start`] prior to this being used.
     pub fn from_registry() -> Addr<Self> {
```

Comment:
Having a `::from_registry()` forces you to import the service type, this variation makes everyone import the `Registry`.  The only argument I'm able to come up is that importing the service type makes it easier to discover in the code where the service is being used, so that would argue for keeping `::from_regsitry()`.  +1 to making it a trait method though.